lib/modules/datasource/helm/index.ts

```diff
@@ -71,6 +71,7 @@ export class HelmDatasource extends Datasource {
           releases: releases.map((release) => ({
             version: release.version,
             releaseTimestamp: release.created ?? null,
+            newDigest: release.digest ?? null,
```

Comment:
```suggestion
            newDigest: release.digest,
```
null isn't allowed on `newDigest` type